2019/OSCON/the-inner-source-learning-path/proposal.md

```diff
@@ -0,0 +1,89 @@
+Proposal to be given via https://conferences.oreilly.com/oscon/oscon-or/user/proposal/propose/cfp/694
+
+# Title
+
+The Inner Source Learning Path
+
+# One-sentence description of your presentation
+
+Learn key roles and principles for effective inner source and how you can share them with others.
+
+# Topic
+
+Sustainable Open Source
+
+# Abstract
+
+Inner source is the application of open source methodologies to company-internal software.
+It's an effective strategy for running shared internal projects and also for developing the people and organizational skills and muscle for wide and deep involvement in open source.
+While simple to define, inner source can be difficult to explain and implement successfully on a broad corporate scale.
+Many participating engineers lack background in open source and the ideas and mechanics of open code development.
+Even those with that background face a new set of constraints and motivations when trying to work openly within an enterprise.
+
+The Inner Source Learning Path (http://bit.ly/inner-source-learning-path) can help.
+It gives a simple, easy-to-understand introduction to inner source.
+Its video segments and written articles introduce key terms, concepts, and principles for effective inner sourcing along with explanations and real-world examples.
+Advanced segments deep dive into explanation of the specific roles of inner source and how to fill them effectively.
+The learning path brings newcomers up to speed and also provides those with experience a common vocabulary and resource to use when discussing inner source.
+
+We'll review the content of the learning path and how you can access it for yourself and share it with others.
+The session includes a(n)
+* short introduction to inner source.
+* explanation of the problems that inner source can solve.
+* overview of the mechanics and roles of inner source.
+* review of the key benefits of inner source.
+* deeper look at the key roles of inner source.
+  * Product Owner
+  * Trusted Committer
+  * Contributor
+* summary of concepts and links to the online learning path resources.
+
```

Comment:
I don‘t know if we‘ll eventually add another LP videos in the future, but wouldn‘t it be nice to call on InnerSource practitioners to share their learnings in a video (on our YouTube Channel) and for newcomers to ask questions they would like to answer? Also, I think this talk would be a great opportunity to outline the other channels we provide to share knowledge: Patterns, Books, Summits. I‘d mention the whole range here.

Reply:
This sounds good to include, but maybe too detailed and loosely coupled to include in this CFP proposal?  Thinking of keeping things simple here so the reviewers can clearly understand the focused thrust of our content and accept it.  Once we're "in" I think that we can include some of those other pieces of information that you bring up in the actual presentation at OSCON.